### PR TITLE
Feature/el03 mott

### PR DESCRIPTION
**Pull request by kbeling4**
Added interface and test for Mott Scattering Correction values.

### CODE REVIEW

File: src/ACEtk/interpretation/EL03/test/MottScatteringCorrection.test.cpp [hunk 1]
```diff
@@ -23,22 +23,22 @@ namespace{
...
+	 "on which the mott scattering correction data points are evaluated") {
       const auto trial = el03.mottScatteringCorrection().energyGrid();
       const auto reference = referenceEnergyGrid | ranges::view::reverse;
       for ( const auto pair : ranges::view::zip( reference, trial ) ) {
 	REQUIRE( pair.first == Approx( pair.second.value ) );
       }
     }
     
-    WHEN("Querying for the Mott scattering correction data"){
+    WHEN("querying for the Mott scattering correction data") {
```
Comment (david8dixon) [inline src/ACEtk/interpretation/EL03/test/MottScatteringCorrection.test.cpp]: Add a when for both energies and note the energy

**Review by david8dixon — APPROVED** [review]
Minor detail - consider renaming ```num_angles``` to something that is more consistent with the actual data i.e. h(theta). Not sure how to refer to this...

**Comment by jlconlin** [comment]
Please keep in mind the [Style Guide](http://www.njoy21.io/DevelopersGuide/Style/) and the [naming conventions](http://www.njoy21.io/DevelopersGuide/Style/Code.html). Variables, classes, namespaces, etc. have a convention so that we can know what each is without having to search through the code.

File: src/ACEtk/interpretation/EL03/MottScatteringCorrection.hpp [hunk 2]
```diff
@@ -0,0 +1,42 @@
+class MottScatteringCorrection {
+  const Table& table;
+
+  auto h() const {
```
Comment (jlconlin) [inline src/ACEtk/interpretation/EL03/MottScatteringCorrection.hpp]: Functions that are more than one logical line long (i.e., more than one semi-colon) need to be in their own file in order to match the style guide.

**Review by apmccartney — COMMENTED** [review]
well done

**Pull request merged**